lib/player.rb

```diff
@@ -0,0 +1,13 @@
+# rubocop:disable Style/GlobalVars
```

Comment:
Please do not disable this rubocop flag. Instead, don't use global variables. This is not a good practice.
What you can do if you need to access the variables is to create an attribute_accessor (reader or writer). Read more about accessors [here](https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby) and [here](https://www.rubyguides.com/2018/11/attr_accessor/).

You can also create a method that returns that variable and that would also be ok.